packages/elements/src/card/__test__/card.test.js

```diff
@@ -110,6 +110,7 @@ describe('card/Card', () => {
 
       setTimeout(() => menuButtonPart.dispatchEvent(new Event('tap')));
       await oneEvent(menuButtonPart, 'tap');
+      await nextFrame(); // required for Safari
```

Comment:
Can you also check here if it really needs `nextFrame` ? Can you replace it with elementUpdated ?

Reply:
I wonder why the `elementUpdateted` is better than `nextFrame` in this case what if element fire an event but not update itself?

Reply:
It is not better but some element only needs to wait for update to complete on the other hand `nextFrame()` does not guarantee that.